package.json

```diff
@@ -61,7 +62,8 @@
     "upload-local-data-files": "node ./scripts/uploadComponent --type local-data-files-updater",
     "upload-ntp-sponsored-images-components": "node ./scripts/uploadComponent --type ntp-sponsored-images",
     "upload-ntp-super-referrer-component": "node ./scripts/uploadComponent --type ntp-super-referrer",
-    "upload-parameter-updates": "node ./scripts/uploadComponent --type client-model-parameters"
+    "upload-parameter-updates": "node ./scripts/uploadComponent --type client-model-parameters",
+    "upload-speedreader": "node ./scripts/uploadComponent --type speedreader-updater"
```

Comment:
uploadComponent.js should know speedreader-updater type

Reply:
fixed, though I don't see `type` option used anywhere within the script anyway

Reply:
You're right. that could be cleaned up :)